scylla-server/migrations/2024-11-10-031516_create_all/down.sql

```diff
@@ -1,4 +1,8 @@
--- This file should undo anything in `up.sql`
-DROP TABLE "run";
+-- Drop Foreign Keys
```

Comment:
why is this file changed?

Reply:
It didn't work at all, and it needed constraints dropped to work